html_css/intermediate_css/more-text-styles.md

```diff
@@ -68,14 +68,24 @@ This method _may_ be more reliable than relying on an third-party font API, but
 You learned the basics of manipulating fonts in our foundations lessons, but there is quite a bit more that you can do with CSS when it comes to manipulating text styles. These rules are all relatively simple and self-explanatory. You can refer to the docs for any questions you might have.
 
 #### font-style
-Typically used to make a font italic. You learned about the HTML `<em>` tag which uses an italic font, but `<em>` _also_ signifies that the text it wraps is significant, or should be emphasized in some way. A good rule of thumb to follow is that if you just want text to be italic (or bold, underlined, highlighted, etc.), use a CSS property. Otherwise if text should have some sort of semantic emphasis, use the correct HTML element. For example, if you want all your header text to be italic you should use `font-style` to accomplish this. If you want some text in the _middle of a sentence_ to appear italic in order to emphasize that text, it is appropriate to use an `em` element.
+Typically used to make a font italic. You learned about the HTML `<em>` tag, which uses an italic font, but `<em>` _also_ signifies that the text it wraps is significant or should be emphasized in some way. A good rule of thumb to follow is that if you just want text to be italic (or bold, underlined, highlighted, etc.), use a CSS property. Otherwise, if text should have some sort of semantic emphasis, use the correct HTML element.
 
+For example, if you want all your header text to be italic you should use `font-style` to accomplish this. If you want some text in the _middle of a sentence_ to appear italic in order to emphasize that text, it is appropriate to use an `em` element. The [MDN doc on the Emphasis Element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/em) puts stress on our point above.
+
+We should use `font-style: italic;` if italics is required for styling purposes.
```

Comment:
The codeblocks are not showing up as intended on the [lesson preview tool](https://www.theodinproject.com/lessons/preview). Adding a new line just before the codeblock should fix it. Can you add that please?